### ISSUE TITLE
Display passed events

### ISSUE THREAD

**Issue by aumiras** [kalebheitzman/grav-plugin-events#17]
I am trying to grasp the grav techniques and unfortunately got stuck. I am doing the website for culture center and it is essential to have some kind of events.
If I am making a collection with 

`content:
    items:
        '@taxonomy':
            type: event`

I am getting only upcoming events. I tried to use 

`content:
    items:
        '@page.descendants': /events`

 and I do get the past events displayed but if I sort them by date

`dateRange:
        start: 1/1/2014
        end: 1/1/2016` 

I am not getting any.
I was hopping for archive plugin to sort it out somehow, but it doesn't show anything as well. 
Probably I am missing something.

**Comment by paulcmal** [comment]
Have you tried enabling "Show past events" in the plugin config?

You can edit this setting from the admin panel, or from grav-plugin-events/events.yaml

**Comment by aumiras** [comment]
Yes, it is enabled.

**Comment by paulcmal** [comment]
What if you try to order by event start instead of date?

```
content:
  items: '@page.descendants': /events
  order: header.event.start
```

**Comment by aumiras** [comment]
It lists events, but then, how to sort them by date and in ascending or descending order? I tried to put another `content:
  items: 
    '@page.descendants': /events
  order: header.event.start
  order:
    by: date
    dir: asc`
but if I change to _desc_ it lists in the same order. 
Sorry for my troubles, I feel that I miss some tutorial part :)

**Comment by paulcmal** [comment]
Did you order by date or by header.event.start?

If your header contained `content: items: '@page.descendants': /events order: header.event.start order: by: date dir: asc`, i.e. two order variables, I guess Grav would default to the first one it reads when parsing the page header, therefore ignoring the dir variable that goes with your second 'order'.

Try this, maybe?

```
content:
  items: '@page.descendants': /events
  order:
    by: header.event.start
    dir: desc
```

**Comment by aumiras** [comment]
Great, but I have other issues now :) 
I get this strange order: May 6, 2016; June 29, 2016; July 25, 2016; July 25, 2016 (the same event doubled); August 25, 2015; March 9, 2014; March 7, 2014; June 5, 2016

**Comment by kalebheitzman** [comment]
@aumiras I think there is a doubling event that I haven't been able to pin down. Please check out https://github.com/kalebheitzman/grav-brandr-pages for proper configuration of events and see if that helps any. I'll continue trying to track down sorting and doubling issues. Thanks!

**Comment by paulcmal** [comment]
I can reproduce the issue on my setup.

I tried messing around a bit and here's what I got:
![Doubling events with an end](https://framapic.org/g58vU2rbXbmQ/yNfJCtOUFk9H.png)

Using `'@page.descendants': /concerts` as item collection (instead of taxonomy type event). The Norwich event was the only one without an end date, and it is the only one not displayed twice. Although it appears to have an end date on the calendar.

The [skeleton](https://github.com/paulcmal/grav-cistemfailure/tree/dc6efb3c79dfdda93e686ab95f4caa85884c0dcb) I used and [how I fixed it](https://github.com/paulcmal/grav-cistemfailure/commit/c51e3f3b35ed550cdfd56e9c7a92118277324310).

Is this of any help @aumiras ?

EDIT: But I've stumbled upon the original problem. Even when enabling "Displaying past events" in the admin interface, I'm only getting upcoming events. See [here](https://dev.subversive.audio/concerts).
